sub_scripts/lxc_build.sh

```diff
@@ -88,7 +88,13 @@ then	# Si le conteneur existe déjà
 fi
 
 echo -e "\e[1m> Création d'une machine debian $DISTRIB minimaliste.\e[0m" | tee -a "$LOG_BUILD_LXC"
-sudo lxc-create -n $LXC_NAME -t debian -- -r $DISTRIB >> "$LOG_BUILD_LXC" 2>&1
+uname=$(uname -m)
+if [[ $uname =~ aarch64 ]]
```

Comment:
Believe `uname -m` should gives you the architecture only, so it would be simply "aarch64", thus, no need of [[ ]] and =~.
I have no such arch on my own to test, but you should try with `if [ "$(uname -m)" == "aarch64" ]`

Reply:
Good point, my bash skill are near 0, I will test that